docs/admin/admission-controllers.md

```diff
@@ -592,7 +592,16 @@ versions >= 1.9).
 ## Is there a recommended set of admission controllers to use?
 
 Yes.
-For Kubernetes >= 1.9.0, we strongly recommend running the following set of admission controllers (order matters for 1.9 but not >1.10):
+
+For Kubernetes >= 1.10.0, we strongly recommend running the following set of admission controllers (order doesn't matter).
+
+Note: ```--admission-control``` was deprecated in 1.10 and replaced with ```--enable-admission-plugins```.
+
+```shell
+--enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota
+```
+
+For Kubernetes >= 1.9.0, we strongly recommend running the following set of admission controllers (order matters).
```

Comment:
This is a little confusing since "For Kubernetes >= 1.9.0 ..." includes 1.10, but you've just added a recommendation right before this "For Kubernetes >= 1.10.0 ..."

Recommend rephrasing something like the following. Note that this recommendation also uses standard language for referring to version ranges (we aren't consistent, but trying to get there when opportunity arises):

"For Kubernetes version 1.10.0 and later, we recommend running ... " (Replacing the parenthetical comment with a complete sentence would be a nice improvement, too.)

and then

"For Kubernetes version 1.9.0 and earlier ..." (but check -- should this be instead "For Kubernetes version 1.9.X and earlier ..." ?)

(While you're at it, we can get rid of "strongly" too -- adds no value.)

UNLESS it's the case that we think it's equally fine to run 1.10.0 and later without `--enable-admission-plugins`. But that's not my impression from related docs ...